src/app/state/index.tsx

```diff
@@ -21,6 +21,7 @@ const STATE_LIST = [DelegationState, DelegationV2State];
 
 export interface AppState {
   availableUTXOs?: UTXO[];
+  allUTXOs?: UTXO[];
```

Comment:
@jrwbabylonlab it's not clear to me why do you use `allUTXOs` instead of `availableUTXOs`?

Reply:
@jrwbabylonlab This is expected behaviour IMO. If user decided to filter out ordinals (to save them) we should not let him to stake his previously created delegation with ordinals